sinks/signalfx/signalfx.go

```diff
@@ -239,19 +239,26 @@ func MigrateConfig(conf *veneur.Config) error {
 	return nil
 }
 
-// CreateSignalFxSink creates a new SignalFx sink for metrics. This function
+// ParseConfig decodes the map config for a SignalFx sink into a
+// SignalFxSinkConfig struct.
+func ParseConfig(config interface{}) (interface{}, error) {
```

Comment:
...building on my comment in server.go, this would then be 

```suggestion
func ParseConfig(config interface{}) (SinkConfig, error) {
```

Reply:
Done.